README.md

```diff
@@ -8,3 +8,10 @@ to trigger a custom deployment pipeline just push the tag with skip
 git tag -a skip:0.1.2 9fceb02 -m "Message here"
 ```
 
+# Creating a new release
+
+```bash
+gh release create -d -F <input-file> -t "<title>" --target <branch_or_commit_hash>
```

Comment:
**docs:** I'm guessing `gh` is the github cli? If it's needed for this, maybe some instructions or a link to where to find it

Reply:
yep let me add that